resources/helm/overlays/istio-telemetry/prometheus/templates/_sd_namespace.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "sdNamespaces" }}
+{{- if gt (len .Values.prometheus.scrapingNamespaces) 0 }}
+namespaces:
+  names:
+{{ toYaml .Values.prometheus.scrapingNamespaces | indent 2 }}
```

Comment:
I think that namespaces must always be set, even empty, when SMCP is deployed in the multi-tenant mode. Otherwise Prometheus will scrape services, endpoints and pods at cluster scope level and will log 2 errors per second - this is huge noise.